engine/apps/user_management/migrations/0025_auto_20241017_1919.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 4.2.15 on 2024-10-17 19:19
+import logging
+
+from django.db import migrations
+import django_migration_linter as linter
+
+logger = logging.getLogger(__name__)
+
+
+def populate_general_log_slack_channel(apps, schema_editor):
+    Organization = apps.get_model('user_management', 'Organization')
+    SlackChannel = apps.get_model('slack', 'SlackChannel')
+
+    logger.info("Starting migration to populate general_log_slack_channel field.")
+
+    queryset = Organization.objects.filter(general_log_channel_id__isnull=False, slack_team_identity__isnull=False)
+    total_orgs = queryset.count()
+    updated_orgs = 0
+    missing_channels = 0
+
+    logger.info(f"Total organizations to process: {total_orgs}")
+
+    for org in queryset:
+        slack_id = org.general_log_channel_id
+        slack_team_identity = org.slack_team_identity
+
+        try:
+            slack_channel = SlackChannel.objects.get(slack_id=slack_id, slack_team_identity=slack_team_identity)
+
+            org.general_log_slack_channel = slack_channel
+            org.save(update_fields=['general_log_slack_channel'])
```

Comment:
I'd propose to use `bulk_update` here

Reply:
updated in https://github.com/grafana/oncall/pull/5191/commits/ce5a0978ccc151dd9b87a32b027a4ce8bf84fb79